app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.environ["API_KEY"]
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL + f"?key={API_KEY}&q={FILTERING}")
+
+    city = response.json().get("location").get("name")
+    country = response.json().get("location").get("country")
+    localtime = response.json().get("location").get("localtime")
+    celsius = response.json().get("current").get("temp_c")
+    condition = response.json().get("current").get("condition").get("text")
```

Comment:
It's a good practice to check the response status code before attempting to parse the JSON. Consider adding error handling to manage unsuccessful requests or JSON parsing errors.

Reply:
Done